### PR TITLE
#159965294 Integrate travisCI with badges.

### PR DESCRIPTION
**Pull request by meshack-mbuvi**
#### What does this PR do?
- Integrate travis CI with a status badge
#### Description of Task to be completed?
- Link repository with Travis 
- Add configuration files for *Travis CI*; *.travis.yml* 
- Add a build status badge to *README.md* file in the repository
#### How should this be manually tested?
- Click [here](https://github.com/andela/ah-magnificent6/tree/ch-github-travisCI-integration-with-badges-159965294) and followed by clicking the build status badge on README.md file which would navigate you to _https://travis-ci.org/andela/ah-magnificent6_
#### What are the relevant pivotal tracker stories?
[Integrate Travis CI with badges](https://www.pivotaltracker.com/story/show/159965294)

**Note:** Build status indicates that *Build unknown* since the badge is linked to develop branch on travis. The status will be updated when this PR is accepted and merged.
.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Authors Haven - A Social platform for the creative at heart.
```
Comment (parseen254) [inline README.md:3]: @meshack-mbuvi any reason for removal of this title?

Comment (meshack-mbuvi) [reply README.md:3]: Seen it. It should be there. Let me correct it.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+[![Build Status](https://travis-ci.org/andela/ah-magnificent6.svg?branch=master)](https://travis-ci.org/andela/ah-magnificent6)
```
Comment (parseen254) [inline README.md]: change this to point to develop or the whole repo as we will rarely touch master branch

File: README.md (hunk 1 above)
Comment (meshack-mbuvi) [reply README.md:3]: @parseendavid Done

File: README.md (hunk 2 above)
Comment (meshack-mbuvi) [reply README.md]: @parseendavid Done

**Review by parseen254 — COMMENTED** [review]
Make sure to set environment variables required by the app... I would like to see a passing build. @meshack-mbuvi

**Comment by meshack-mbuvi** [comment]
@parseendavid Done

**Comment by micahoriaso** [comment]
Nice work @meshack-mbuvi . Just resolve the conflicts now.

**Comment by pabishai** [comment]
@meshack-mbuvi  The build is passing.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (parseen254) [inline .vscode/settings.json]: @meshack-mbuvi remove this file from history.

**Review by parseen254 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json (hunk 3 above)
Comment (meshack-mbuvi) [reply .vscode/settings.json]: @parseendavid Your request is completed.

**Pull request merged**